pages/item/_itemId/index.vue

```diff
@@ -0,0 +1,124 @@
+<template>
+  <v-container fluid class="pa-0 itemContainer">
+    <v-img
+      :src="getItemBackdrop(item.Id)"
+      class="d-flex align-end backdropImage"
+      max-width="100%"
+    >
+      <div class="d-flex flex-wrap itemDetailsContainer">
+        <div class="itemDetailsLeft">
+          <v-img
+            v-if="item.ImageTags && item.ImageTags.Logo && getAspectRatio() > 1"
+            :src="getImageLink(item.Id, 'Logo')"
+            contain:alt="item.Name"
+            max-width="50%"
```

Comment:
Logos usually have a max size of 800x310 (Per fanart.tv guidelines). We should ideally avoid enlarging them, as it looks pixelated or blurry.

Reply:
This shouldn't enlarge the image, just ensure that it's max width is 50% if it is on a small resolution display. See Below:
<details>
<summary>Without max-width 50%</summary>

![image](https://user-images.githubusercontent.com/18101008/92589262-38c80100-f292-11ea-93a2-f773ae447afc.png)

</details>

<details>
<summary>With max-width 50%</summary>

![image](https://user-images.githubusercontent.com/18101008/92589323-4da49480-f292-11ea-891e-49fcf1a74d7b.png)

</details>